src/main/scala/com/amazon/deequ/comparison/ComparisonResult.scala

```diff
@@ -17,5 +17,11 @@
 package com.amazon.deequ.comparison
 
 sealed trait ComparisonResult
-case class ComparisonFailed(errorMessage: String) extends ComparisonResult
-case class ComparisonSucceeded() extends ComparisonResult
+
+case class ComparisonFailed(errorMessage: String, ratio: Double = 0) extends ComparisonResult
+case class ComparisonSucceeded(ratio: Double = 0) extends ComparisonResult
```

Comment:
We should keep 1 set of states. Right now, there are 2 sets of states. `Comparison[Failed/Succeeded]` vs `DataSynchronization[Failed/Succeeded]`. Having too many states can result in confusion for the end user.

Reply:
I was planning, to consolidate once `ReferentialIntegrity` is also integrated into verification suite. I will handle that in next PR.